stock_request/security/stock_request_security.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.module.category" id="module_category_stock_request">
+        <field name="name">Stock Request</field>
+        <field name="parent_id" ref="base.module_category_warehouse_management"/>
+        <field name="sequence">10</field>
+    </record>
+
+    <record id="group_stock_request_user" model="res.groups">
+        <field name="name">Stock Request User</field>
+        <field name="implied_ids" eval="[(4, ref('base.group_user'))]"/>
+        <field name="category_id" ref="module_category_stock_request"/>
+    </record>
+
+    <record id="group_stock_request_manager" model="res.groups">
+        <field name="name">Stock Request Manager</field>
+        <field name="users" eval="[(4, ref('base.user_root'))]"/>
```

Comment:
Functional review (first attempt):
I'm only able to access to Stock Request General Settings if I use Become a SuperUser, i think problem is that now admin user is defined as base.user_admin